01-main/manifest

```diff
@@ -153,6 +153,7 @@ ocenaudio
 tezos-baking
 onedriver
 onlyoffice-desktopeditors
+ookla-speedtest
```

Comment:
The CI test fails because `ookla-speedtest` is not found : the name we use (in manifest and the package filename) must match the app exactly. You need to use `speedtest`